Makefile

```diff
@@ -0,0 +1,335 @@
+	--mount type=bind,source=$(DOCKER_MOUNT_CONFIG_PATH)/tradesv3.sqlite,target=/freqtrade/tradesv3.sqlite \
+	--mount type=bind,source=$(DOCKER_MOUNT_CONFIG_PATH)/tradesv3.dryrun.sqlite,target=/freqtrade/tradesv3.dryrun.sqlite
+
+# App image filter
+APP_IMG_FILTER := docker ps -aq --filter ancestor=$(APP_CONTAINER) --format="{{.ID}}"
+
+## Build App container
+DOCKER_RUN_OSX := docker run -it --rm -e TZ=`ls -la /etc/localtime | cut -d/ -f8-9` 
+DOCKER_RUN_NIX := docker run -it --rm \
+	--mount type=bind,source=$(DOCKER_MOUNT_CONFIG_PATH)/localtime,target=/etc/localtime,readonly
+
+docker-app: build-docker-app
+build-docker-app: setup-create
+	@docker build -t $(APP_CONTAINER) -f $(APP_DOCKERFILE) .
+
+## Start app including START_ARGS
+docker-start-bot: docker-app
+ifeq ($(OSFLAG), OSX)
+	@$(DOCKER_RUN_OSX) -d --name $(APP_CONTAINER)-bot \
+		$(DOCKER_MOUNTS)\
+		$(APP_CONTAINER) $(START_ARGS)
+else 
+	$(DOCKER_RUN_NIX) -d --name $(APP_CONTAINER)-bot \
+		$(DOCKER_MOUNTS)\
+		$(APP_CONTAINER) $(START_ARGS)
+endif
+
+## open a shell in the container
+docker-shell:
+ifeq ($(OSFLAG), OSX)
+	@$(DOCKER_RUN_OSX) --name $(APP_CONTAINER)-shell -entrypoint=/bin/bash \
+		$(DOCKER_MOUNTS)\
+		$(APP_CONTAINER) -i
+else
+	@$(DOCKER_RUN_NIX) --name $(APP_CONTAINER)-shell --entrypoint=/bin/bash \
+		$(DOCKER_MOUNTS)\
+		$(APP_CONTAINER) -i
+endif
+
+# Run historical backtest
+TS := $(shell /bin/date "+%Y-%m-%d---%H-%M-%S")
+backtest:
+ifeq ($(OSFLAG), OSX)
+	@$(DOCKER_RUN_OSX) -d --name $(APP_CONTAINER)-backtest \
+		$(DOCKER_MOUNTS)\
+		$(APP_CONTAINER) $(START_ARGS) freqtrade backtesting --timerange=20180131- \
+		--refresh-pairs-cached  --export trades \
+		--export-filename=user_data/backtest_data/backtest_$(DEFAULT_STRATEGY)_$(TS).json
+else
+	@$(DOCKER_RUN_NIX) -d --name $(APP_CONTAINER)-backtest \
+		$(DOCKER_MOUNTS)\
+		$(APP_CONTAINER) $(START_ARGS) backtesting --timerange=20180131-\
+		--refresh-pairs-cached  --export trades \
+		--export-filename=user_data/backtest_data/backtest_$(DEFAULT_STRATEGY)_$(TS).json
+endif
+
+# Run backtest on live exchange
+backtest-live:
+ifeq ($(OSFLAG), OSX)
+	@$(DOCKER_RUN_OSX) -d --name $(APP_CONTAINER)-backtest-live \
+		$(DOCKER_MOUNTS)\
+		$(APP) backtesting --live --export trades \
+		--export-filename=user_data/backtest_data/backtest_$(DEFAULT_STRATEGY)_$(TS).json
+else
+	@$(DOCKER_RUN_NIX) -d --name $(APP_CONTAINER)-backtest-live \
+		$(DOCKER_MOUNTS)\
+		$(APP) backtesting --live --export trades \
+		--export-filename=user_data/backtest_data/backtest_$(DEFAULT_STRATEGY)_$(TS).json
+endif
+
+## Stop app without losing data  
+docker-stop:
+	@-$(APP_IMG_FILTER) | grep -q . && docker stop $$($(APP_IMG_FILTER)) || true
+
+## Remove all containers built from app image
+docker-clean: docker-stop
+	@-$(APP_IMG_FILTER) | grep -q . && docker rm $$($(APP_IMG_FILTER))
+
+## Show status of Docker containers 
+docker-state:
+	@docker ps $($(APP_IMG_FILTER))
+
+## Delete all compiled Python files
+pyclean:
```

Comment:
Just a note; I personally use `py.cleanup -p` (from pycmd) for this, but I don't think this additional dependency should be added into requirements-dev.txt...